Bugfix/UI loading

bruno-fs opened the pull request:
closes #2159

Fixes a bug preventing UI to be properly rendered.

It also includes an integration test making sure all UI resources are loaded properly so this won't happen again.

## Why was this happening
UI assets were not placed in the correct folder. Instead of placing them on `/app/quipucords/quipucords/template` and `/app/quipucords/client` they were placed at  `/app/quipucords/quipucords/template/template` and `/app/quipucords/client/client`
The tests added on this PR are making sure HTML templates, stylesheets and JS scripts can be properly loaded.

## Steps to reproduce and context
added to the issue #2159

## How to verify this works
Build quipucords from this branch and repeat steps to reproduce (see #2159) replacing `quay.io/quipucords/quipucords:1.0.0` with the name or id of the image built.

gtanzillo reviewed (approved):
The changes look good to me @bruno-fs. Can you add a few details to the description about the issue this is fixing and how QE can verify it?

bruno-fs commented:
> The changes look good to me @bruno-fs. Can you add a few details to the description about the issue this is fixing and how QE can verify it?

Ok! Updated and moved more context to an bug issue. Can you check if this is OK?

gtanzillo commented:
Thanks for adding the additional details, @bruno-fs! LGTM 👍
